### PR TITLE
Add flag to dump all work data for a matcher graph

### PR DESCRIPTION
**Pull request by jamieparkinson**
This is useful for debugging merging issues!

### CODE REVIEW

**Comment by alexwlchan** [comment]
I would be careful of calling this "source"; we have another script that gets the original records from Miro/Sierra/whatever – https://github.com/wellcomecollection/catalogue-pipeline/blob/main/scripts/get_source_records.py

Maybe "--save-works"?

**Comment by jamieparkinson** [comment]
Good point, fiddly language

**Review by kenoir — APPROVED** [review]

File: pipeline/matcher/scripts/get_graph.py [hunk 1]
```diff
@@ -56,7 +58,12 @@ def main(index_date, work_id):
 
     node_ids = [node["id"] for node in graph_component]
     node_links = [node["linkedIds"] for node in graph_component]
-    nodes = get_nodes_properties(es, index_date=index_date, work_ids=node_ids)
+    nodes = get_nodes_properties(
+        es,
+        index_date=index_date,
+        work_ids=node_ids,
+        fetch_complete_source=emit_work_data,
```
Comment (alexwlchan) [inline pipeline/matcher/scripts/get_graph.py:65]: Change the name of this parameter to use "work"?

**Review by alexwlchan — COMMENTED** [review]
I

**Pull request merged**